### ISSUE TITLE
AWS Identity Store support for create user and group

### ISSUE THREAD

**Issue by antonioned** [hashicorp/terraform-provider-aws#26598]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

AWS docs now state that Identity Store has new API calls for CreateUser and CreateGroup. Hopefully this will be supported in newer aws provider versions.

### New or Affected Resource(s)

CreateGroup
DeleteGroup
CreateUser
DeleteUser

### References

https://docs.aws.amazon.com/singlesignon/latest/IdentityStoreAPIReference/API_Operations.html

(Edited by @justinretzolk to add task list)

- [x] #26606
- [x] #26607

**Comment by bschaatsbergen** [comment]
Picking up the group work 👍🏼

**Comment by jordancparker** [comment]
Hi community,

Would there be scope to expand and include another task to utilise:
- CreateGroupMembership
- DeleteGroupMembership

Which would enable the ability to add aws_identitystore_user to a aws_identitystore_group, appreciate this request would be blocked by the availability of the above tasks.

**Comment by drAlberT** [comment]
Any reasonable ETA?

**Comment by ewbankkit** [comment]
Closed via https://github.com/hashicorp/terraform-provider-aws/pull/26948 and https://github.com/hashicorp/terraform-provider-aws/pull/26674.
